frontend/lib/norent/homepage.tsx

```diff
@@ -1,19 +1,364 @@
+                <br />
+                <p>
+                  Tenants in covered properties are also protected from
+                  eviction, fees, penalties, and other charges related to
+                  non-payment by the CARES Act (Title IV, Sec. 4024) enacted by
+                  Congress on March 27, 2020.
+                </p>
+                <br />
+                <p>
+                  Along with my neighbors, I am organizing, encouraging, and/or
+                  participating in a tenant organization so that we may support
+                </p>
+              </div>
+              <div className="jf-letter-preview-fadeout" />
+            </article>
+          </div>
+        </div>
+      </section>
+
+      <section className="hero has-background-white-ter">
+        <div className="hero-body">
+          <div className="container jf-tight-container has-text-centered jf-space-below">
+            <h2 className="title is-spaced">How it works</h2>
+            <p className="subtitle is-size-4">
+              We make it easy to weigh your options and notify your landlord by
+              email or by certified mail for free.
+            </p>
+            <br />
+          </div>
+          <div className="container jf-wide-container jf-how-it-works-container">
+            <div className="columns is-variable is-8-desktop">
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("chatbox")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    8 Steps
+                  </p>
+                  <p>
+                    Answer a few questions about yourself and your landlord or
+                    management company.
+                  </p>
+                </div>{" "}
+              </div>
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("connection")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    Legal Protections
+                  </p>
+                  <p>
+                    Our letter cites the most up-to-date legal ordinances that
+                    protect tenant rights in your state.
+                  </p>
+                </div>
+              </div>
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("letters")}
+                    alt="Chat boxes"
```

Comment:
Again, set alt text to empty string.

Since all of these are 128x128 and just differ in `src` I wonder if it would help to have a separate one-off component for it?  Since this gets expanded to five frickin' lines by prettier... up to you tho.